### PR TITLE
Show empty state when a session has no vaccinations to record

### PR DESCRIPTION
**Pull request by benilovj**
## What's changed

1. Made the "record vaccinations" index page two-thirds width, to bring closer to the designs
2. Introduced an empty state when there are no vaccinations to record. This will come in handy when tabs are introduced.

image

## Questions for reviewers

1. There's some inline CSS – should I extract it into the CSS files? If so, what could I name the class?
2. Does the proposed microcopy for the empty state make sense?

### CODE REVIEW

File: app/views/layouts/_empty_list.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div style="margin-top: 32px; border: 2px solid #b1b4b6; min-height: 300px">
```
Comment (tvararu) [inline app/views/layouts/_empty_list.html.erb:1]: Could be `.app-empty-list`. Also instead of 32px, you can use the nhsuk-spacing helper

Comment (benilovj) [reply app/views/layouts/_empty_list.html.erb]: @tvararu I've made the suggested changes in [a328455](https://github.com/nhsuk/record-childrens-vaccinations/pull/273/commits/a328455b389d8051ffb4072351a3a5fc808d3017)

File: app/assets/stylesheets/_empty-list.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.app-empty-list {
+  margin-top: nhsuk-spacing(6);
+  border: 2px solid #b1b4b6;
```
Comment (tvararu) [inline app/assets/stylesheets/_empty-list.scss:3]: Does this look ok? Would be more consistent with the NHSUK Design System.

```suggestion
  border: 2px solid $nhsuk-border-color;
```

**Review by tvararu — APPROVED** [review]

File: app/assets/stylesheets/_empty-list.scss (hunk 2 above)
Comment (benilovj) [reply app/assets/stylesheets/_empty-list.scss]: 🤷 but I've changed to the design system default until we have a good reason to deviate...

**Pull request merged**